src/Symfony/Component/Notifier/Bridge/MicrosoftTeams/README.md

```diff
@@ -0,0 +1,22 @@
+Microsoft Teams Notifier
+========================
+
+Provides [Microsoft Teams](https://www.microsoft.com/en/microsoft-365/microsoft-teams/free) integration through Incoming Webhook for Symfony Notifier.
+
+DSN example
+-----------
+
+```
+MICROSOFT_TEAMS_DSN=microsoftteams://default/HOSTPATH
+```
+
+where:
+ - `PATH` has the following format: `/webhook/{uuid}@{uuid}/IncomingWebhook/{id}/{uuid}`
```

Comment:
HOSTPATH or PATH?

Also beware of extra slashes.

Reply:
Fixed